src/main.rs

```diff
@@ -105,21 +158,30 @@ impl FlashAlgo for NumWorksAlgo {
 
     fn erase_sector(&mut self, addr: u32) -> Result<(), ErrorCode> {
         qpi_command(Command::WriteEnable);
+
         let qspi = unsafe { &(*QUADSPI::ptr()) };
+
+        qspi.abr.write(|w| unsafe { w.bits(addr) });
+
+        // TODO: Why doesn't this work with address bytes?
```

Comment:
I have no idea why this doesn't work when using `admode` instead of `abmode`. The command is skipped completely if I change this back.